### ISSUE TITLE
"feature `resolver` is required" issues

### ISSUE THREAD

**Issue by palfrey** [hoodie/notify-rust#114]
See https://github.com/hoodie/notify-rust/runs/3625748281?check_suite_focus=true

```
Error: failed to parse manifest at `/home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/zvariant_derive-2.8.0/Cargo.toml`
Caused by:
  feature `resolver` is required

  this Cargo does not support nightly features, but if you
  switch to nightly channel you can add
  `cargo-features = ["resolver"]` to enable this feature
Error: The process '/usr/share/rust/.cargo/bin/cargo' failed with exit code 101
```

This is due to `zvariant_derive` using the [Cargo resolver feature from 1.51](https://blog.rust-lang.org/2021/03/25/Rust-1.51.0.html#cargos-new-feature-resolver). You can fix this either by upgrading the version of Rust you build with, or reducing the version requirements on that to something that maybe doesn't use that (haven't looked TBH).

**Comment by hoodie** [comment]
oh boy, thanks
I guess I'll have to correct my MSRV to 1.51 now

**Comment by hoodie** [comment]
ought to be fixed with #115 

thanks for pointing this out
